### DISCUSSION TITLE
Is kohsuke java library supported for GitHub Enterprise Server (On Prem)?

### DISCUSSION THREAD

**Discussion by Chandrakant8086** [hub4j/github-api#1236] [Q&A]
Hi,
My questions is if we are using GitHub Enterprise Server (On Prem) then kohsuke java library (https://github-api.kohsuke.org/) is supported or not ?

**Comment by bitwiseman** [comment]
@Chandrakant8086 
This library should work with GitHub Enterprise, though it is not tested against any specific versions of GHE.

**Comment by abhi-malhotra** [comment]
I have used this library on Github Enterprise and it works.
